### ISSUE TITLE
RStudio docker image: startup.sh error message: -e not found

### ISSUE THREAD

**Issue by aryoda** [ppc64le/build-scripts#131]
I have just built the image for RStudio from

https://github.com/ppc64le/build-scripts/blob/master/rstudio/Dockerfiles/v1.2.1335/Dockerfile

and tried to start a container with

`sudo docker run -i -p 8787:8787 rstudio`

but I cannot connect to RStudio Server via the browser with localhost:8787
because the docker container shows this error message:

`/startup.sh: 1: /startup.sh: -e: not found`

The reason seems to be that the first line of the script contains an "-e".

This may possibly be caused by this line in the run part of the Dockerfile

`echo -e '#!/bin/bash\ncd /usr/sbin\nrstudio-server start' >> startup.sh && \
`

**Comment by aryoda** [comment]
I forgot to mention that I am using Fedora release 31 (Thirty One) on an IBM Power9 8-core CPU (revision 2.3 pvr 004e 1203) with kernel version 5.4.13-201.fc31.ppc64le and Xfce as Desktop.

`docker version` says:

```
Client:
Version: 18.09.8
API version: 1.39
Go version: go1.13beta1
OS/Arch: linux/ppc64le
Experimental: false

Server:
Engine:
Version: 18.09.8
API version: 1.39 (minimum version 1.12)
etc. (same as client)
```

**Comment by seth-priya** [comment]
@sarveshtamba is working on this

**Comment by sarveshtamba** [comment]
Fixed and tested. Raised PR at https://github.com/ppc64le/build-scripts/pull/137

**Comment by sarveshtamba** [comment]
@aryoda can you please try to build RStudio docker image from the updated Dockerfile now and let us know the result at your end?

**Comment by aryoda** [comment]
@sarveshtamba Thanks for fixing the issue. I am really sorry but I cannot test the fix currently since my power9 hardware has a defect and a replacement will take about two months due to the required RMA procedure and the complicated customs clearance (I am privately sponsoring the power9 support of R so I cannot just buy another machine).

I will try to find somebody else via a forum post...

**Comment by sarveshtamba** [comment]
@aryoda sure, do let us know in case you need any further assistance.

**Comment by gerrith3** [comment]
@aryoda is there any chance that you can test on the systems at Oregon State University's Open Source Lab:  https://osuosl.org/services/powerdev/ to do a build/test in the interim?

**Comment by ghatwala** [comment]
@aryoda  - any comments on above ?

**Comment by aryoda** [comment]
Oh, forgive me my late feed-back. What a pity nobody followed my request for testing in the forum I have asked for help :-(

I personally wanted to test the fix on exactly the same hardware/OS/app layer constellation
but I am still in the RMA phase of my hardware and delivery bottle-necks do affect my new hardware (current estimation is 4 - 6 more weeks until I can get my replacement hardware).

Thanks for the link to the Oregon State University Open Source Lab initiative. I think testing there would not add value to the tests you have already done so I prefer to wait for my hardware (where the already installed software stack is waiting on the SSD to be reapplied).

So I propose to publish the fix ASAP for now.

I will come back for a retest as soon as I get my new hardware - promised :-)

**Comment by aryoda** [comment]
And: I can close this issue if you want - I will retest it  anyhow

**Comment by seth-priya** [comment]
Thanks @aryoda for reporting and following up on this, since we were able to replicate, fix and test the issue on our side, I am closing this for now, Please feel free to re-open or create another issue if you still find problems in your testing, thank you!
